components/x11/ftgl/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Harry Liebel <hliebel@gmail.com>
+#
+
+PREFERRED_BITS=64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		ftgl
+COMPONENT_VERSION=	2.1.3.5
+HUMAN_VERSION=		2.1.3~rc5
+COMPONENT_FMRI=		library/x11/ftgl
+COMPONENT_PROJECT_URL=	https://sourceforge.net/projects/ftgl
+COMPONENT_SUMMARY=	OpenGL FreeType fonts rendering library
+COMPONENT_DESCRIPTION=	FTGL is a free cross-platform Open Source C++ library that uses Freetype2 to simplify rendering fonts in OpenGL applications. FTGL supports bitmaps, pixmaps, texture maps, outlines, polygon mesh, and extruded polygon rendering modes.
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(HUMAN_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-2.1.3-rc5.tar.gz
+COMPONENT_ARCHIVE_HASH= \
+  sha256:5458d62122454869572d39f8aa85745fc05d5518001bcefa63bd6cbb8d26565b
+COMPONENT_ARCHIVE_URL= \
+  $(COMPONENT_PROJECT_URL)/files/FTGL%20Source/$(HUMAN_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE=	MIT
+COMPONENT_CLASSIFICATION= System/X11
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+CONFIGURE_OPTIONS += --with-x
+CONFIGURE_ENV += LIBS="-lm"
+
+build:		$(BUILD_64)
```

Comment:
Why do we build only 64-bit version?

Reply:
Hi.
I thought all apps were now preferred as 64bit. If so, then do we expect to build 32bit apps against this library in future?
Thanks

Reply:
We are mostly providing both 32/64 bit libraries, as 32-bit applications are still supported.